prefixer/fix-carbon.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+// throw exception if anything fails
```

Comment:
I think this is a temporary fix, isn't it? Could you please leave a comment on what needs to happen for this file to be removed?

Reply:
Only ternary operator fixes are temporary (until new phpScoper with updated parser is released). Cleanups (added in later commit) are not temporary. I'll add a comment to the ternary operators fix and create a Jira for updating PHPScoper.